### PR TITLE
Resolve build scan summary dependencies

### PR DESCRIPTION
**Pull request by sorin-florea**
The dependencies of the `build-scan-summary` library will be resolved from maven central.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -30,33 +30,53 @@ repositories {
...
+        project.objects.named(TargetJvmEnvironment.STANDARD_JVM)
+    )
+}
 
 dependencies {
     argbash("argbash:argbash:2.10.0@zip")
     mavenComponents(project(":configure-gradle-enterprise-maven-extension"))
     mavenComponents("com.gradle:gradle-enterprise-maven-extension:1.18.4")
     mavenComponents("com.gradle:common-custom-user-data-maven-extension:1.13")
+    buildScanSummaryComponent("com.gradle.develocity:build-scan-summary:$buildScanSummaryVersion")
```
Comment (erichaagdev) [inline build.gradle.kts:60]: Should the group be `com.gradle` like the other things we publish? I see the `build-scan-summary` is published under the `com.gradle.develocity` group too.

File: build.gradle.kts [hunk 2]
```diff
@@ -30,33 +30,53 @@ repositories {
...
 allprojects {
     version = releaseVersion.get()
 }
 
 val argbash by configurations.creating
 val mavenComponents by configurations.creating
+val buildScanSummaryComponent by configurations.creating {
+    attributes.attribute(
+        TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE,
+        project.objects.named(TargetJvmEnvironment.STANDARD_JVM)
```
Comment (erichaagdev) [inline build.gradle.kts:51]:
```suggestion
       objects.named(TargetJvmEnvironment.STANDARD_JVM)
```

File: build.gradle.kts [hunk 3]
```diff
@@ -30,33 +30,53 @@ repositories {
         }
     }
     mavenCentral()
+    maven("file://${layout.projectDirectory.dir("components/develocity").asFile.path}")
```
Comment (erichaagdev) [inline build.gradle.kts:33]: Can we add a content exclusion so Gradle does not attempt to resolve `build-scan-summary` from Maven Central?

https://ge.solutions-team.gradle.com/s/m6invvi5asxg2/performance/network-activity#network-requests-all-0

**Review by erichaagdev — CHANGES_REQUESTED** [review]
This is really clever. Have some feedback though.

**Review by jthurne — APPROVED** [review]
I agree with Eric's comments, otherwise LGTM.

File: build.gradle.kts (hunk 1 above)
Comment (sorin-florea) [reply build.gradle.kts:60]: Done

File: build.gradle.kts (hunk 3 above)
Comment (sorin-florea) [reply build.gradle.kts:33]: Done

**Pull request merged**